### DISCUSSION TITLE
Feature suggestion : Print

### DISCUSSION THREAD

**Discussion by sarrek** [minbrowser/min#1621] [Ideas]
Hello,

its possible support for print function.
Just need a option who dont open a new tabs, juste select the printer
enter and the ticket is printed.

Thanks

**Comment by Syndamia** [comment]
If I understood your suggestion correctly, there is already a way to print the whole page. Pressing `Ctrl + P` or clicking the `Top Left Menu (three dots)/File/Print` will open the printing dialogue with which you should be able to print the **whole** page.

![image](https://user-images.githubusercontent.com/46843671/120272182-4d7f7580-c2b5-11eb-992d-534961412cc1.png)
